packages/sdk/src/swap/sdk.ts

```diff
@@ -112,7 +107,11 @@ export class SwapSDK {
         ttl: 60_000,
       },
       networkStatus: {
-        fetch: () => this.trpc.networkStatus.query(),
+        fetch: async () => {
+          const res = await this.apiClient.networkStatus();
+          assert(res.status === 200, 'Failed to fetch network status');
```

Comment:
maybe we want to include the response in the error, not sure if we will see the reason in sentry otherwise